app/topics/shared/topic.model.ts

```diff
@@ -77,7 +97,7 @@ export class Topic {
     let inAMonth = new Date();
     inAMonth.setDate(inAMonth.getDate() + 30);
     return new Topic(-1, '', '', 'InProgress', null, new Array<User>(),
-      null, '', '', inAMonth.toISOString(), new Date().toISOString(), null, parentTopics);
+        null, '', '', inAMonth.toISOString(), new Date().toISOString(), null, parentTopics, new Array<number>(), new Array<number>(), new Array<number>());
```

Comment:
But I thinks it's a task for the constructor to create these empty arrays from the empty User Arrays. It's mostly a consistency concern. Doing so, you can create a Topic with student alpha, but have students ids 4 and 5. So to guarantee consistency the constructor should create an Array of ids from the original User Array, even if it's empty